tigerpath/majors_and_certificates/majors/CLA_2018.json

```
@@ -0,0 +1,373 @@
+{
+  "type": "Major",
+  "name": "Classics",
+  "code": "CLA",
+  "degree": "AB",
+  "year": 2018,
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-classics",
+    "https://classics.princeton.edu/programs/undergraduate/tracks"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Michael A. Flower",
+      "email": "mflower@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Joshua Billings",
+      "email": "jhbillings@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Tracks",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": null,
+      "req_list": [
+        {
+          "name": "Classics",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": null,
+          "req_list": [
+            {
+              "name": "Prerequisites",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "To enter this program, a student normally should have completed CLG 108 or LAT 108 or demonstrated intermediate proficiency in Greek or Latin through test scores (SAT, AP), a placement exam, or coursework. A strongly motivated student who has completed CLG 102 or LAT 102 (or CLG 103 or LAT 103) may concentrate, with permission of the departmental representative.",

```

Since the prerequisite can also count as a departmental, it should be set to be allow to double-counting (that is, it can count for this requirement _and_ another requirement. Let me know if I'm misreading.
```suggestion
              "explanation": "To enter this program, a student normally should have completed CLG 108 or LAT 108 or demonstrated intermediate proficiency in Greek or Latin through test scores (SAT, AP), a placement exam, or coursework. A strongly motivated student who has completed CLG 102 or LAT 102 (or CLG 103 or LAT 103) may concentrate, with permission of the departmental representative.",
              "double_counting_allowed": true,
```